src/test/java/systemtests/ClearListCommandSystemTest.java

```diff
@@ -40,13 +40,14 @@ public void clear() {
     private void assertCommandSuccess(String command) {
         Model expectedModel = new ModelManagerStub();
```

Comment:
Should probably be using `getModel()`

Reply:
But doesn't matter tbh

Reply:
In this specific case might actually be better to use ModelManagerStub since we are checking that the list is cleared.